roaster-restassured/src/main/java/hu/icellmobilsoft/roaster/restassured/helper/HealthCheckTestHelper.java

```diff
@@ -51,27 +51,54 @@ public class HealthCheckTestHelper {
 
     /**
      * Testing /health
-     * 
-     * @param baseUri
-     *            URI for health endpoint
+     *
+     * @param baseUri URI for health endpoint
      */
+    @Deprecated(since = "0.8.0", forRemoval = true)
```

Comment:
guytudom maga /health attol meg valid vegpont marad, meg ha van is bontasa masik 3-ra https://download.eclipse.org/microprofile/microprofile-health-4.0/microprofile-health-spec-4.0.html#_appendix_a_rest_interfaces_specifications

Reply:
igen, valid marad, teszt szempontjából viszont csak 1-re kellene lőni és használatban is nem az agregált /health-et kellene megcélozzuk, hanem konkrétan azt, amit tesztelni akarunk vele, ezért jelöltem meg deprecated-re

Reply:
megbeszéltük, leszedtem róla a deprecated megjelölést